components/layout_2020/display_list/mod.rs

```diff
@@ -322,12 +322,25 @@ impl<'a> BuilderForBoxFragment<'a> {
                 )
             };
             let b = fragment.style.get_border();
-            wr::BorderRadius {
+            let mut r = wr::BorderRadius {
                 top_left: corner(&b.border_top_left_radius),
                 top_right: corner(&b.border_top_right_radius),
                 bottom_right: corner(&b.border_bottom_right_radius),
                 bottom_left: corner(&b.border_bottom_left_radius),
+            };
+            // Normalize radii that add up to > 100%.
```

Comment:
Maybe paste the specification text so it's clear where this calculation comes from:

> Let f = min(Li/Si), where i ∈ {top, right, bottom, left}, Si is the sum of the two corresponding radii of the corners on side i, and Ltop = Lbottom = the width of the box, and Lleft = Lright = the height of the box. If f < 1, then all corner radii are reduced by multiplying them by f.